tools/self-hosted-runner-utils/README.md

```diff
@@ -0,0 +1,45 @@
+# self-hosted-runner-utils
+
+This is a collection of utilities to help facilitate our self hosted infrastructure for Github Actions,
+these are meant to be general purpose and can be re-used for other projects wishing to have some level
+of self hosted infra utilities.
+
+## Dependency installation
+
+Dependencies for these utils are found in the requirements.txt, you can install using:
+
+```
+pip install -r requirements.txt
+```
+
+## Formatting
+
+Tools here are formatted with black, use the `Makefile` to format your code:
+
+```
+make format
+```
+
+## clear_offline_runners.py
+
+This is a utility to clear offline self hosted runners. The reason why this may be necessary is if your
+scale down lambda does not always clear up self hosted runners on the Github side, so this is useful for
```

Comment:
Do we know why? [the dashboard](https://metrics.pytorch.org/d/LFSW__inz/lambda-status?orgId=1&from=1632689809419&to=1632776209419&viewPanel=13) seems to show a pretty high ratio of errors / invocations

Reply:
This was most likely due to the lambda timeout, so this may be a one time use script but I think it should be fine to have this utility around just in case we need to use it again